scoring/cscore.metta

```diff
@@ -44,6 +44,34 @@
            (() (println! (PenalizedScore: $updatedPScore))))
         
             (mkCscore $scor $cpxy $cpxyPenalty $uniformityPenalty $updatedPScore)))
+;converts a score to an expression
+; (: cScoreToExpr (-> Cscore Expression))
+(= (cScoreToExpr (mkCscore $scor $cpxy $complexityPenalty $uniformityPenalty $penalizedScore))
+    (let* (
+        ($e1 (cons-atom $penalizedScore ()))
+        ($e2 (cons-atom $uniformityPenalty $e1))
+        ($e3 (cons-atom $complexityPenalty $e2))
+        ($e4 (cons-atom $cpxy $e3))
+        ($e5 (cons-atom $scor $e4))
+    )
+        ($e5)
+    )
+)
+
+; (: exprToCScore (-> Expression Cscore))
+(= (exprToCScore $exp)
```

Comment:
Same here. Since you seem to be certain as to the shape of the expression you can use the same structure as the above